packages/firestore/rollup.config.exp.release.js

```diff
@@ -0,0 +1,114 @@
+/**
```

Comment:
What do we need the non-release rollup configs for?  I am getting rid of the  one dependency I have on the non-mangled builds (https://github.com/firebase/firebase-js-sdk/pull/3440). Are there others?

Reply:
Replaced the non-release config with the release config and removed the release rollup config files.